auth/api/iam/api.go

```diff
@@ -335,18 +338,24 @@ func (r Wrapper) OAuthClientMetadata(ctx context.Context, request OAuthClientMet
 
 	return OAuthClientMetadata200JSONResponse(clientMetadata(*identity)), nil
 }
-func (r Wrapper) PresentationDefinition(_ context.Context, request PresentationDefinitionRequestObject) (PresentationDefinitionResponseObject, error) {
+func (r Wrapper) PresentationDefinition(ctx context.Context, request PresentationDefinitionRequestObject) (PresentationDefinitionResponseObject, error) {
 	if len(request.Params.Scope) == 0 {
 		return PresentationDefinition200JSONResponse(PresentationDefinition{}), nil
 	}
 
-	// todo: only const scopes supported, scopes with variable arguments not supported yet
-	// todo: we only take the first scope as main scope, when backends are introduced we need to use all scopes and send them as one to the backend.
-	scopes := strings.Split(request.Params.Scope, " ")
-	presentationDefinition := r.auth.PresentationDefinitions().ByScope(scopes[0])
-	if presentationDefinition == nil {
+	authorizer, err := did.ParseDID(request.Did)
```

Comment:
should we check whether the node "owns" the DID, to potentially unburden a REST API backend that queries the XIS' database?

Reply:
'Normal' operation would not be optimised, just unsanctioned access. A generic security assessment on the 'open' APIs would be a better approach. (IsOwned goes to Vault...)